File: Ruby/0213.rb
def convert_to_title(column_number)
  result = ''
  while column_number > 0
    column_number -= 1
    result = ('A'.ord - 1 + column_number % 26).chr + result
    column_number /= 26
  end
  result
end

# 範例
puts convert_to_title(1)    # Output: "A"
puts convert_to_title(28)   # Output: "AB"
puts convert_to_title(701)  # Output: "ZY"

File: Python/0213.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        result = ''
        while columnNumber:
            columnNumber -= 1
            result = chr(ord('A') + columnNumber % 26) + result
            columnNumber //= 26
        return result

# 簡單範例
sol = Solution()
print(sol.convertToTitle(1))    # Output: "A"
print(sol.convertToTitle(28))   # Output: "AB"
print(sol.convertToTitle(701))  # Output: "ZY"